runtime/src/main/java/org/corfudb/runtime/view/stream/BackpointerStreamView.java

```diff
@@ -193,6 +193,21 @@ protected boolean fillReadQueue(final long maxGlobal,
                     }
                 }
 
+                // If hole filling is disabled, we will retry forever.
+                if (runtime.isHoleFillingDisabled()) {
+                    int i = 0; // For exponential backoff.
+                    while (currentEntry.getType() == DataType.EMPTY) {
+                        currentEntry =
+                                runtime.getAddressSpaceView().read(currentRead);
+                        try {
+                            Thread.sleep(1 << i);
+                            i++;
+                        } catch (InterruptedException ie) {
+                            throw new RuntimeException(ie);
```

Comment:
If an interrupted exception occurs, some entity (perhaps via a signal) is requesting the exception be propagated so that graceful termination can occur.